ontology/epso.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: ontology_detail
+id: EPSO
+label: Epilepsy Ontology
+title: Epilepsy Ontology
+description: A application driven Epilepsy Ontology sith official terms from the ILAE.
+domain: disease
+tracker: https://github.com/phwegner/Epilepsyontology/issues
+dependencies:
+  - id: BFO
+products:
+  - id: epso.owl
+  - id: EPSO_merged.owl
+contact:
+  email: alpha.tom.kodamullil@scai.fraunhofer.de
+  label: Alpha Tom Kodamullil
+  github: akodamullil
+  
+repository: git@github.com:phwegner/Epilepsyontology.git
```

Comment:
should be https://github.com/phwegner/Epilepsyontology

Reply:
@matentzn changed it